### PR TITLE
Fix consistency in language strings

### LINKED ISSUE #6341: Comment about consistency in Language strings
Just want to comment on the use of `{0}` in the strings: 
there are multiple ways of showing for example filenames in the messages:
`"{0}"`, `({0})`, `` `{0}` `` or simply without quotes or parenthesis `{0}`

Should maybe be standardized to one format?

### PR DESCRIPTION
**Pull request by tangix**
**Description**
Fixes #6341
Supersedes #6414

Fixed consistency in Language strings as discussed in #6414 
I hope this PR is correctly branched from 4.3

**Checklist:**
- [X] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [X] Conforms to style guide

### CODE REVIEW

**Review by kenjis — APPROVED** [review]
Thank you!

**Review by paulbalandan — APPROVED** [review]

**Comment by tangix** [comment]
Darn - I forgot to update the Change Log! Should I update this branch and push again?

**Comment by MGatner** [comment]
@tangix Yes please do!

File: user_guide_src/source/changelogs/v4.3.0.rst [hunk 1]
```diff
@@ -116,6 +116,7 @@ Changes
 - Config
     - All atomic type properties in ``Config`` classes have been typed.
     - Changed the default setting to not redirect when a CSRF check fails so that it is easy to see that it is a CSRF error.
+- Updated English language strings to be more consistent
```
Comment (kenjis) [inline user_guide_src/source/changelogs/v4.3.0.rst:119]:
```suggestion
- Updated English language strings to be more consistent.
```

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
Thank you!

**Pull request merged**